torch/testing/_internal/common_methods_invocations.py

```diff
@@ -405,7 +405,9 @@ def method_tests():
         ('ceil', (S, S, S), NO_ARGS, '', (True,)),
         ('ceil', (), NO_ARGS, 'scalar', (True,)),
         ('rad2deg', (S, S, S), NO_ARGS),
+        ('rad2deg_', (S, S, S), NO_ARGS),
```

Comment:
You can remove this and the the deg2rad_ entry. I don't think method_tests supports inplace ops, and we don't need to add tests for these inplace operators here.